system/include/emscripten/emscripten_mmap.h

```diff
@@ -0,0 +1,54 @@
+#ifndef EMSCRIPTEN_MMAP_H_
+#define EMSCRIPTEN_MMAP_H_
+
+#include <math.h>
+
+#ifndef EMSCRIPTEN
+#define define_mmap_write(T, name) \
```

Comment:
Please use `__EMSCRIPTEN__` instead of `EMSCRIPTEN`, since that is old deprecated form.